test/unit/vl-proza-message.test.html

```diff
@@ -193,9 +192,9 @@
         });
 
         test('het aanpassen van het data-vl-code attribuut zal ervoor zorgen dat het bericht voor die code getoond wordt', () => {
-            fetchMock.mock('proza/domein/foo/foo', {code: "foo", tekst: "foofoo"});
-
             const proza = fixture('vl-proza-messages-fixture');
+            fetchMock.restore();
```

Comment:
Zouden we dit niet in een after/before zetten?

Reply:
Een restore zit al in de `teardown` functie, maar door het inladen van het proza element gebeuren er al HTTP calls, die het resultaat van de testen verstoren. Deze calls worden na het inladen gereset, zodat deze 2 testen niet meer flaky zijn.